OBSIDIAN/user.css

```
@@ -0,0 +1,95 @@
+    transition: opacity 0.6s ease;
+}
+
+/* The animation image */
+#mytheme-splash-img {
+    width: 260px;
+    height: 260px;
+    object-fit: contain;
+}
+
+@font-face {
+    font-family: 'FamiljenGrotesk';
```

Stylelint (17.14.0)

[error] Expected no quotes around "FamiljenGrotesk" (font-family-name-quotes)

(font-family-name-quotes)